### ISSUE TITLE
ID attribute not highlighted if it comes first in script tag

### ISSUE THREAD

**Issue by dflock** [daaain/Handlebars#58]
This code, with the `id` attribute first, doesn't highlight properly:
```
<script id="data_categories" type="text/x-handlebars-template">
```

but it does if you swap the `id` and `type` attributes around, eg:

![screenshot - 15-05-06 - 05 10 31 pm](https://cloud.githubusercontent.com/assets/47756/7506294/e8436144-f412-11e4-894f-177e2274dc9b.png)

**Comment by daaain** [comment]
Right, I had a dig and this is a side effect of handling these different `text/template` script types and recognising the Handlebars templates inside them (rather than just treating them as generic script). So basically whatever is between the script tag name and the `type` attribute gets picked up as the beginning of the capture group.

I'm not really sure how to work around this :( if you have some time to play around with it the overly greedy regex is in this line: https://github.com/daaain/Handlebars/blob/master/Handlebars.JSON-tmLanguage#L597
